packages/studio/src/components/PillPicker/PillPickerInput.tsx

```diff
@@ -0,0 +1,103 @@
+import { useCallback } from "react";
+import { ReactComponent as X } from "../../icons/x.svg";
+import classNames from "classnames";
+import PillPicker from "./PillPicker";
+
+const pillContainerClass =
+  "flex flex-wrap min-h-[38px] items-center border border-gray-400 rounded-lg pt-2 pb-1 px-2 w-full text-sm";
+
+interface PillPickerInputProps {
+  selectedItems: string[] | undefined;
+  availableItems: string[] | undefined;
```

Comment:
I think it might be simpler to have an items prop like `items: { value: string, displayName: string, isSelected: boolean }[]`? Otherwise it's a little hard to understand how all 3 of these props interact

Reply:
I think a single `items` prop would simplify some stuff, but it's also slightly more restrictive than having separate props. For example, if a developer has specified classes in a Tailwind class prop that aren't part of a user's custom Tailwind config, I think the current approach would still show the classes as expected and allow them to be removed, but not added again from the dropdown. By combining the props into a single `items` prop, we'd no longer be able to support developer-specified selected items that come from outside of the list of items that are shown in the dropdown.

We could do an intermediate approach, where `selectedItems` and `availableItems` have type `{ value: string, displayName?: string }`, and still support the above case. But, this adds complexity to `TailwindPropInput`, which doesn't need the concept of display names. We'd have to map from string to object and back, whereas a separate prop for the display name makes it easier to "opt out" of that complexity.